src/compiler/scala/tools/nsc/javac/JavaParsers.scala

```diff
@@ -800,16 +800,8 @@ trait JavaParsers extends ast.parser.ParsersCommon with JavaScanners {
       val superclazz =
         AppliedTypeTree(javaLangDot(tpnme.Enum), List(enumType))
       val finalFlag = if (enumIsFinal) Flags.FINAL else 0l
-      val abstractFlag = {
-        // javac adds `ACC_ABSTRACT` to enum classes with deferred members
-        val hasAbstractMember = body exists {
-          case d: DefDef => d.mods.isDeferred
-          case _         => false
-        }
-        if (hasAbstractMember) Flags.ABSTRACT else 0l
-      }
```

Comment:
OK, can you add the same comment as in the classfile parser (https://github.com/scala/scala/blob/2.12.x/src/compiler/scala/tools/nsc/symtab/classfile/ClassfileParser.scala#L545-L546), or a reference to it?